setup.py

```diff
@@ -7,15 +7,15 @@
 # NOTE: When updating botocore make sure to update awscli/boto3 versions below
 install_requires = [
     # pegged to also match items in `extras_require`
-    'botocore>=1.32.4,<1.33.2',
```

Comment:
I missed this in the previous review, this was dangerous as there were changes to aiobotocore needed to match the latest botocore, so I'm not sure they were all backwards compatible

Reply:
There were no backwards incompatible changes in that previous version range.